src/config/model.rs

```diff
@@ -115,7 +115,9 @@ impl Config {
     }
 
     fn path_from_xdg_alt() -> Result<PathBuf> {
-        let path = env::var("HOME").chain_err(|| "Cannot find `HOME` env var")?;
+        let path = env::var("HOME")
+            .or(env::var("USERPROFILE"))
+            .chain_err(|| "Cannot find `HOME` env var")?;
```

Comment:
`USERPROFILE` is only used by Windows, right? So I will go with sth more concise, like:

```rust
let home_var = if cfg!(target_family = "windows") {
    "USERPROFILE"
} else {
    "HOME"
};
let home_path: PathBuf = env::var(home_var)
    .chain_err(|| format!("Cannot find `{}` env var", home_var))?
    .into();
```

See https://doc.rust-lang.org/reference/conditional-compilation.html#target_family.